src/qt/transactiontablemodel.cpp

```diff
@@ -95,25 +95,27 @@ class TransactionTablePriv
      */
     QList<TransactionRecord> cachedWallet;
 
-    bool fQueueNotifications = false;
+    bool m_loaded = false;
+    int m_progress = 100;
```

Comment:
It would be nice to add doxygen comments for new data members.

Reply:
Added.